### ISSUE TITLE
Issues with Participate#Submission

### ISSUE THREAD

**Issue by carlinmack** [hfg-gmuend/openmoji-website#57]
## https://openmoji.org/styleguide/#submission

1. This should detail the workflow for inkscape (at least) and affinity designer.
2. This should contain the same information as https://github.com/hfg-gmuend/openmoji/blob/master/CONTRIBUTING.md or should link here

**Comment by b-g** [comment]
Hi @carlinmack, Yes!

1. yes! Could you add the export infos for inkscape (and affinity)?
2. yes! But I would only link to CONTRIBUTING.md e.g. "How to contribute for developers", as the reality is that a lot of designers struggle with Git. We deliberately didn't put all the Github stuff under /styleguide ... as the threshold should be as low as possible to contribute. And the Pros know the Github anyway.
3. ?? The entire page is the styleguide ...

**Comment by carlinmack** [comment]
3. I know, that's why it shouldn't be called "Participate" haha, I think Participate could be a subheading on this page and have sections for designers (what design work we need) and developers (link to CONTRIBUTING.md)

**Comment by b-g** [comment]
> I know, that's why I don't think it shouldn't be called "Participate" haha, I think Participate could be a subheading on this page and have sections for designers (what design work we need) and developers (link to CONTRIBUTING.md)

Still not sure whether I understand. I like that the title in the navigation is participate (and not styleguide) as this underpins IMO better that everyone can actually participate, whereas as styleguide is normally something static. Hence I would leave the page title as it is.

However it is a good call to make the subtitles more explicit. Maybe instead of "Participate" a "Contribute" section, or dedicated sections for Illustrator, Inkscape and Affinity?

**Comment by carlinmack** [comment]
I also like having Participate in the header but the first thing you should see on the page should be how to participate. Right now users are shown the grid we use which isn't very informative.

**Comment by b-g** [comment]
Yes, your are right. Maybe we could start with a general paragraph and then all the styleguide sections and conclude how to contibute / submit etc.?

**Comment by carlinmack** [comment]
Yep that sounds good!

**Comment by dnlutz** [comment]
I'm aware that there is some work lurking for me in this area ; )
In the near future, I would structure the "Participate" page like this:

**Intro / Call for participation**
- OpenMoji needs You
- Please contribute the missing Unicode 13 Emoji

**Design: Basics**
- Sizes, Colors, Line Widths, Shadows …

**Design: Things to Do**
- Examples for not yet perfect Emoji and how to improve them
- this will also be a new styleguide chapter

**Technical Basics**
- Curve quality, Anchor points, Layer Structure, Do's and Dont's …

**Contribution**
- Export and production from various sources
- Maybe we will even accept neatly designed (illustrator) source files?

**Comment by b-g** [comment]
@dnlutz ok :) This will be quite a restructuring ... this seems rather realistic towards spring (Feb 2020). 

Maybe we could in the meantime improve it slightly:
- Starting with a nice short general text on "how to and why to participate". As @carlinmack said ... currently not optimal to start with "Grid"
- Add submission instructions for Inkscape and Affinitiy
- Add prominent link to CONTRIBUTING.md

**Comment by carlinmack** [comment]
I can do the instructions for Inkscape/Affinity sometime after the 13th as finals week is kicking in, most of my contributions will be very minimal until then

**Comment by b-g** [comment]
Great + NP + Thanks! Same for me :) Good luck for your final weeks!

**Comment by carlinmack** [comment]
Inkscape instructions were done in 30e8879

**Comment by b-g** [comment]
@carlinmack Maybe another quick win might be a first "greeting" paragraph ... instead of the not so warm welcome with the grid section?

**Comment by carlinmack** [comment]
Here's what I have so far, 
![image](https://user-images.githubusercontent.com/6676843/73199063-25e92d80-4102-11ea-8fb3-22fd4ad87ac0.png)

**Comment by carlinmack** [comment]
![image](https://user-images.githubusercontent.com/6676843/73215551-4a550200-4122-11ea-814c-6ebc509c982c.png)
![image](https://user-images.githubusercontent.com/6676843/73215553-4aed9880-4122-11ea-8aaf-55b75416bcab.png)

**Comment by b-g** [comment]
@carlinmack Nice!! Comments:

- Would move emoji 13 up as top prio and link a Github issue with a list
- Big headline has to really align left ... otherwise all HfG people go crazy in terms of "good" typography :)
- Big headline text ... maybe better?  "Designer? Developer? Opensource Enthusiast? Everyone Is Welcome to Participate!" 
- All titles should be Title Case
- "Improve design of emojis" > "Improve Quality of OpenMoji"
- "Help design new emoji" > "Help Design New OpenMojis"

What do you think?

**Comment by bohnacker** [comment]
@carlinmack: Cool. Great work on the website!
@b-g: Agree with all of it. I don't really like Title Case but maybe that's a personal thing :-)

**Comment by carlinmack** [comment]
1. 👍
2. 👍 I am not trained in typography, I'll do as you say haha. btw it's the styling from the front page "OpenMoji is an open source project of 53 students..."
3. I think this is a good compromise "Designer? Developer? Love Open Source? Everyone Can Participate!"
4. Agree with bohnacker, not sure
5. and 6. I prefer "Emoji"/"Emojis" in both cases. For a catchy tagline I think we should stick with ideas that people are more likely to already care about. I think designers are fundamentally interested in improving emoji in general whereas as they probably don't care about improving OpenMoji (yet). 

Please criticise my "emoji 13.0", haven't been trained to do good kerning haha. I think the stroke is too eye catching and dominates the heading.

**Comment by b-g** [comment]
3. 👍 "Designer? Developer? Love Open Source? Everyone Can Participate!"
4. I'm fine either way ... and I'm not attached to Title Case. But it has to be consistent over the entire Styleguide, which currently is Title Case. Either we normalize everything or we stick with Title Case. All Good For Me.
5. / 6. OK 👍 for "Emoji" and "Emojis" ... your are the native speaker :)
7. Image "emoji 13.0" is fine for me too ... Daniel can tweak it if he wants :)

**Comment by carlinmack** [comment]
Opened PR #72

**Comment by dnlutz** [comment]
The call for participation for emoji 13 is important.But we should definitely wait until we have improved guidelines, a step-by-step design and technical tutorial AND an improved illustrator file to start with. I know this is a lot of work but we have to do it before we publish our call. I'm sorry that at the moment I am still too busy with other projects but an end is foreseeable. 

@carlinmack / @b-g We have to work together on this topic: i see emoji 13 as an opportunity to significantly improve the quality of our emoji. At the moment OpenMoji contains many very good emoji but also some mediocre and even bad ones that are not consistent with the rest.

In addition to the guidelines we also have to think about how to find comrades-in-arms outside Github. Many designers are not on Github, so that's a serious issue.

**Comment by carlinmack** [comment]
So to do:

- [ ] improved guidelines
- [ ] a step-by-step design tutorial 
- [ ] a step-by-step technical tutorial 
- [ ] an improved illustrator file to start with
- [ ] how to find comrades-in-arms outside Github

What do you mean with the fourth item? 

Hopefully there is some /r/programmingrequests for graphic design we can ask

**Comment by dnlutz** [comment]
Thank you for the list! The fourth item is more a question than a to-do: How could we motivate experienced designers / illustrators to participate in OpenMoji? These people often don't use GitHub (or they struggle with it like myself ; )

Irrespective of this: it will take some time to process the list. So it would be great if you could update the intro of the participate page. The layout and the content is good, it's much more motivating than just the grid.

The only element I would reconsider is the emoji with the big eyes (> improve design of emoji). The eyes are not typical for the guidelines. If you want you can use this polar bear or maybe you find another good example. I think it would be good to not only show smileys.

![polar_bear](https://user-images.githubusercontent.com/38292227/73459766-b67c7380-4377-11ea-81d6-8992a25a6fd1.png)

**Comment by carlinmack** [comment]
Sorry I meant what do you mean with this "an improved illustrator file to start with"

**Comment by dnlutz** [comment]
ah ok … A file that already contains some basic shapes / components / examples might be helpful for newbies. It would be easier to use these components to build something new.

**Comment by carlinmack** [comment]
Oh I see that would be good, I do a lot of finding similar emojis and using their assets so this would be very helpful!

additionally the openmoji font is currently only distributed as a illustrator file, I'll update the website with a plain SVG version

**Comment by carlinmack** [comment]
I would add that we need to:

- [ ] Add information for what work we need from developers. 

There's stuff with OpenMoji font for sure, but we should have a to do list

**Comment by b-g** [comment]
Hi @carlinmack! 

On mobile the styleguide lacks a separation between our prompt ("please help us") and where the styleguide actually starts. I think an additional headline "Styleguide" or any other separator element would be helpful, or? 

You can check it with your phone on staging:
https://hfg-gmuend.github.io/openmoji-website/styleguide/

**Comment by carlinmack** [comment]
Yeah I think a headline that says Styleguide would be good, however we use `<h1>`s for the Grid, Contour etc headings already so it would look like this 

# Designer? Developer?
# Love Open Source?
# Everyone Can Participate!

### Help Design New Emoji
The next set of emojis are due to be released...

### Improve Consistency with Other Platforms
To be useful for communication, emoji need to...

### Improve Quality of Emojis
Design is subjective but we want our emoji to...

# Styleguide
# Grid
The Grid serves as orientation for the size of...
### Basic Forms

**Comment by b-g** [comment]
Well we could introduce a gentle line (like Github does often) eg.:

Designer? Developer?
Love Open Source?
Everyone Can Participate!
----------------------------------------------
content
...

Styleguide
----------------------------------------------
content
...

**Comment by carlinmack** [comment]
yeah that would work nicely 👍

**Comment by carlinmack** [comment]
should I do this?

**Comment by b-g** [comment]
@carlinmack I won't say no! :) Would be wonderful!

**Comment by carlinmack** [comment]
done 4c90779

**Comment by b-g** [comment]
@carlinmack Great + Many thanks!!

**Comment by carlinmack** [comment]
I think we should make a new issue/s with the currently outstanding items

**Comment by cemoon95** [comment]
Hello, I'm a straggler in 2026 adding a comment!
Unfortunately, even as of this writing, Affinity is not a very cooperative program in terms of exporting SVGs, and I think it would still be impossible to have it natively export SVGs that are truly agnostic as per the OpenMoji project's needs.

If you absolutely have to, in the best-case scenario, you can reduce your headache by exporting with these settings:

![Image](https://github.com/user-attachments/assets/64a6e3ff-8a69-426d-a139-ebfbac971bde)
![Image](https://github.com/user-attachments/assets/e84b52b3-9066-473c-b2d0-c4fc0e402e44)

...but even then, you'll still need to re-import it into Inkscape to make sure everything checks (there's a high possibility some stroke settings may have gotten distorted) and export from there, and even _then_ you'll still need to add id="emoji" manually with Edit > XML Editor before you do.

![Image](https://github.com/user-attachments/assets/d2d6d30d-4c17-4d89-8b14-b50de33c1d7c)

Unfortunately, Affinity's dev team doesn't seem to be very keen on doing much about its SVG handling because it's aimed more at general graphic designers than web devs 😓 I wouldn't blame you for just leaving it off or adding a note telling people to use Inkscape instead, but if you do want to add it, I'm leaving this here in case it helps.
